app/pom.xml

```diff
@@ -163,6 +171,12 @@
             <artifactId>apicurio-registry-utils-tests</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.tngtech.keycloakmock</groupId>
+            <artifactId>mock</artifactId>
+            <scope>test</scope>
+            <version>0.4.0</version>
+        </dependency>
```

Comment:
Interesting.

Reply:
Removed since we don't need to mock Keycloak. We've added properties based auth for testing purposes.